index.bs

```diff
@@ -372,12 +375,24 @@ To run the <dfn>bounce tracking timer</dfn> algorithm given a [=moment=] on the
 [=wall clock=] |now|,
 perform the following steps:
 
+1. [=map/For each=] |host| -> |activationTime| of [=user activation map=]:
+    1. If |activationTime| + [=bounce tracking activation lifetime=] is before
+        |now|, then [=map/remove=] |host| from [=user activation map=] and
+        [=bounce tracking grace period=].
+
+        Note: We clear both the entries in [=user activation map=] and
+        [=candidate bounce tracking map=] when a user activation expires.  This
+        is done so that we only consider deletion for the site once after the
+        potential bounce tracking event.  A user activation the later expires
+        won't cause site deletion long after the potential tracking event
+        occurred.  This is intended to reduce unexpected state loss for users;
```

Comment:
I think the fact that the entry is removed from the candidate bounce tracking map before it's checked for presence in the user activation map, is enough to prevent sites' data from being deleted "long after the potential tracking event". As long as the activation lifetime is longer than the bounce tracking grace period, and the process that adds to the candidate bounce tracking map checks the activation map, I think it also can't happen that the site is removed from the activation map before the activation map has a chance to save the site from deletion. (It's not terrible to delete from both maps here; I'm just making sure the explanation of why is correct.)

Reply:
I haven't written the part that adds entries to the bounce tracking map.  My intent was to add it to the map unconditionally there because it matches our implementation.  But some of our implementation choices are driven by things unrelated to the spec (metrics, honoring browser history clearing tools, etc) and we could switch to not adding to the candidate map if there is an activation present.

Reply:
I dropped the description and switched it to an assert instead.